superset/utils.py

```diff
@@ -513,6 +513,18 @@ def table_has_constraint(table, name, db):
     return False
 
 
+def upload_file(file):
+    """Takes in the name of the file to be uploaded and uploads that file"""
+
+    from superset import app
+    config = app.config
+    if not file or not file.filename:
+        raise Exception("No file chosen for upload")
+    file.save(os.path.join(config['UPLOAD_FOLDER'], file.filename))
+    if file.filename not in os.listdir(config['UPLOAD_FOLDER']):
```

Comment:
Can this happen in practice, i assume file.save will return or raise something in case of error

Reply:
True. Taken it out and left it to the exception that will be raised if something goes wrong.